monitoring/google/cloud/monitoring_v3/proto/uptime_pb2.py

```diff
@@ -1232,12 +1357,6 @@
           minimum of 3 locations must be provided, or an error message
           is returned. Not specifying this field will result in uptime
           checks running from all regions.
-      is_internal:
-          If this is true, then checks are made only from the
-          'internal\_checkers'. If it is false, then checks are made
-          only from the 'selected\_regions'. It is an error to provide
-          'selected\_regions' when is\_internal is true, or to provide
-          'internal\_checkers' when is\_internal is false.
```

Comment:
This is still being removed rather than deprecated. Checking with folks working on other languages to see if there's been contact with the monitoring team